composer.json

```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": ">=8.1",
```

Comment:
This is quite the jump and with Xero updating API responses server side that are validated against an hardcoded list client side (the last one being the identifier of the subscription plans) please consider keeping this range very wide. Or remove the hardcoded validations to avoid breaking existing installs on sites that haven't yet been able to run on later versions of php.

Reply:
I would suggest bumping to support php version 7.1 as a minimum as that will allow you to use nullable parameters which you will need to get past errors like

```  
Deprecated: XeroAPI\XeroPHP\Models\Accounting\Item::__construct(): Implicitly marking parameter $data as nullable is deprecated, the explicit nullable type must be used instead in /var/www/vendor/xeroapi/xero-php-oauth2/lib/Models/Accounting/Item.php:254.
```
on php 8.4 - see 
https://wiki.php.net/rfc/deprecate-implicitly-nullable-types

and

https://www.php.net/manual/en/migration71.new-features.php